other/materials_designer/create_point_defect.ipynb

```diff
@@ -3,7 +3,7 @@
   {
```

Comment:
<p>Should be cleaned</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Exabyte-io/api-examples/pull/136/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='6'/>